cluster/gce/config-test.sh

```diff
@@ -241,6 +241,17 @@ SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
 # Optional: install a default StorageClass
 ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 
+# Optional: Enable legacy ABAC policy that makes all service accounts superusers.
+if [[ "${E2E_UPGRADE_TEST:-}" == "true" ]]; then
```

Comment:
I am pretty sure e2e-runner.sh leaks this var, but this is not intentional. I plan to delete e2e-runner,.sh and its magical global state and replace it with flags that the job explicitly uses to call kubetest.

Could we solve this by adding `ENABLE_LEGACY_ABAC=false` to the .env file for the relevant upgrade jobs?